site/views/examples/share.njk

```diff
@@ -20,59 +20,91 @@ aside:
   }
 </style>
 
-<section>
-  <h2>Share example</h2>
+<section class="row">
+  <div class="col-12">
+    <uip-root>
+      <script type="text/html"
+              label="Default share button"
+              uip-snippet
+              uip-snippet-js="js-snippet-share">
+        <h4>Share button with default icon rendering</h4>
+        <esl-share-button name="linkedin" default-icon></esl-share-button>
+      </script>
 
-  <h3>Share button examples</h3>
-  <p>Example of share button with default icon rendering.</p>
-  <esl-share-button name="linkedin" default-icon></esl-share-button>
-  <hr>
+      <script type="text/html"
```

Comment:
maybe we can merge this with the previous tab? call it "Share button" and add the example without the icon that was there before? leave all 4 examples

![image](https://github.com/user-attachments/assets/4d76533b-f136-41c9-adc9-66e2be23face)

Reply:
@abarmina I like this idea, thanks